example/WORKSPACE

```diff
@@ -1,62 +1,25 @@
-### RULES_PYTHON ###
-
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-RULES_PYTHON_NAME = "rules_python"
-
-RULES_PYTHON_TAG = "0.4.0"
-
-RULES_PYTHON_SHA = "954aa89b491be4a083304a2cb838019c8b8c3720a7abb9c4cb81ac7a24230cea"
-
-RULES_PYTHON_REPO = "bazelbuild"
-
-RULES_PYTHON_ARCHIVE = "tar.gz"
-
-RULES_PYTHON_URL = "https://github.com/{repo}/{name}/releases/download/{tag}/{name}-{tag}.{archive}".format(
-    name = RULES_PYTHON_NAME,
-    archive = RULES_PYTHON_ARCHIVE,
-    repo = RULES_PYTHON_REPO,
-    tag = RULES_PYTHON_TAG,
-)
-
-# use http_archive rule to load rules_python repo
+### RULES_PYTHON ###
 http_archive(
-    name = RULES_PYTHON_NAME,
-    sha256 = RULES_PYTHON_SHA,
-    url = RULES_PYTHON_URL,
+    name = "rules_python",
+    url = "https://github.com/bazelbuild/rules_python/releases/download/0.5.0/rules_python-0.5.0.tar.gz",
+    sha256 = "cd6730ed53a002c56ce4e2f396ba3b3be262fd7cb68339f0377a45e8227fe332",
 )
```

Comment:
This requires you to change the version in two places now, but ok, I see that it's easier to look at, especially for new users, so I'm okay with that.